### PR TITLE
feat: allow hydrating single file

### PR DESCRIPTION
**Pull request by Niktux**

### CODE REVIEW

**Comment by lebris** [comment]
Looks good, but I'm disappointed, there is no *unicorn*, *pony* nor *burger* is these unit tests :disappointed:

**Comment by Niktux** [comment]
Even in L56 ? :stuck_out_tongue_winking_eye:

File: src/Karma/Application.php [hunk 1]
```diff
@@ -117,7 +118,18 @@ private function initializeSourceFileSystem()
...
             foreach($paths as $path)
             {
-                $adapter->mount($path, new Local($path));
+                if(is_file($path))
+                {
+                    $filename = basename($path);
+                    $path = realpath(dirname($path));
+                    $localAdapter = new SingleLocalFile($filename, new Local($path));
+                }
+                else
```
Comment (Niktux) [inline src/Karma/Application.php]: Can avoid else

**Comment by lebris** [comment]
you're right, my bad :smile:

**Pull request merged**